examples/custom-charset/pages/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Head from 'next/head'
+export default class Index extends React.Component {
+  static async getInitialProps ({res}) {
+    let contentType = res._headers['content-type']
+    let charSet = contentType ? contentType.slice(contentType.indexOf('=') + 1, contentType.length) : 'utf-8'
```

Comment:
This will cause every client-side page change to change the charSet to utf-8

Reply:
Yeah, I did that on purpose because as referenced on line 16 of the [head library](https://github.com/zeit/next.js/blob/canary/lib/head.js) utf-8 is the default value used anyway. I thought that it would be more useful for the developer to change the default value here rather than it silently change in the background.

I'm not opposed to getting rid of it and throwing an error if no value is created server-side though. Let me know what you think.

Reply:
@sergiodxa Any thoughts on previous comment?

Reply:
@rickyplouis since this is an advanced case I'd just hardcode the value, making the example as simple as possible.